hi-ml-azure/src/health_azure/utils.py

```diff
@@ -2020,8 +2019,14 @@ def check_config_json(script_folder: Path, shared_config_json: Path) -> Generato
         else:
             raise ValueError("Either a shared config.json must be present, or all 3 environment variables for "
                              "workspace creation must exist.")
+
+
+@contextmanager
+def check_config_json(script_folder: Path, shared_config_json: Path) -> Generator:
```

Comment:
Missing docstring. It's not clear to me why this is being separated out?

Reply:
Oh good catch, I forgot to rebase it, I was just experimenting